docs/en/operation/Metrics-System.md

```diff
@@ -23,13 +23,13 @@ following instances are currently supported:
 * Master: The Alluxio master process.
 * Worker: The Alluxio worker process.
 
-Each instance can report to zero or more sinks.
+Each instance can report to zero or more sinks, found in `$ALLUXIO_HOME/core/common/src/main/java/alluxio/metrics/sink/`.
```

Comment:
Because this is user documentation, we don't expect users to have the anything under `./**/src**/` because they will be working with a distribution tarball. I would either point them to a direct github link like: https://github.com/Alluxio/alluxio/tree/master/core/common/src/main/java/alluxio/metrics/sink